operate/config/docker-compose.identity.yml

```diff
@@ -113,7 +113,7 @@ services:
       - CAMUNDA_OPERATE_ZEEBE_GATEWAYADDRESS=zeebe:26500
       - CAMUNDA_OPERATE_ELASTICSEARCH_URL=http://elasticsearch:9200
       - CAMUNDA_OPERATE_ZEEBEELASTICSEARCH_URL=http://elasticsearch:9200
-      - SPRING_PROFILES_ACTIVE=dev-data,identity-auth
+      - SPRING_PROFILES_ACTIVE=dev-data
```

Comment:
🔧 I think for 8.8 branches and newer this docker-compose may not make much sense. We could validate if we can likely remove the entire file to remove ambiguity :)